tests/www/test_views.py

```diff
@@ -521,6 +521,8 @@ def test_dag_stats(self):
     def test_task_stats(self):
         resp = self.client.post('task_stats', follow_redirects=True)
         self.assertEqual(resp.status_code, 200)
+        self.assertEqual(set(resp.json().items()[0][1][0].keys()),
```

Comment:
The below might fix it

```suggestion
        self.assertEqual(set(resp.json.items()[0][1][0].keys()),
```

Reply:
Both are fixed. Thanks @kaxil